setup_dev_env.sh

```diff
@@ -28,6 +28,7 @@ export MONGO_HOST=$LH
 
 export QL_HOST=$LH
 export CRATE_HOST=$LH
+export CRATE_AUTH_HOST=$LH
```

Comment:
i had a quick look at the whole changes:
i would not duplicate the configuration auth / no auth, this makes test longer for no reason.
so let's stick just add auth to the already deployed crate.

wdyt?

Reply:
I thought running the tests twice, with "normal" crate and with crate with authentication enabled, would be better to ensure the changes have no impact in environments with unauthenticated crates and to make it kind of easier to debug (if the test fails in crate and crate-auth, probably missing permissions is not the reason for this. if it works in crate but fails in crate-auth, it may be related to permissions). And additionally to make it easier to run the tests with older crate version (pre 4.5, where authentication was an enterprise feature), in that case the cratedb version env just needs to be set and the crate-auth tests deselected.

But i'm not sure if this is enough benefit to justify the longer test run